cms/djangoapps/contentstore/asset_storage_handlers.py

```diff
@@ -714,7 +715,15 @@ def get_asset_json(display_name, content_type, date, location, thumbnail_locatio
     Helper method for formatting the asset information to send to client.
     '''
     asset_url = StaticContent.serialize_asset_key_with_slash(location)
-    external_url = urljoin(configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL), asset_url)
+
+    ## .. filter_implemented_name: LMSPageURLRequested
+    ## .. filter_type: org.openedx.course_authoring.lms.page.url.requested.v1
+    lms_root, _ = LMSPageURLRequested.run_filter(
+        url=configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
+        org=location.org,
+    )
```

Comment:
For the sake of consistency, I'd prefer if we also returned the organization. Please, refer to these examples for the needed modifications:
https://github.com/openedx/edx-platform/blob/9e55223dbfd76b5566aef23507459bd913646b6f/openedx/features/course_experience/__init__.py#L112-L116
https://github.com/openedx/edx-platform/blob/9e55223dbfd76b5566aef23507459bd913646b6f/lms/djangoapps/learner_home/serializers.py#L266-L269

The same comment applies to the other filter.

Reply:
@mariajgrimaldi I don't understand what you want to do. Do you refer to something as `lms_root, org = LMSPageURLRequested.run_filter`? If the answer is yes, I think we have a problem, because `org` is not used in the code. Unless we leave it with pylint disable in that line

Reply:
Yes, I was referring to doing something like:

```
course_org = course.id.org
lms_root, _ = LMSPageURLRequested.run_filter(
    url=configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
    org=course_org,
)
```

Although you're right, we shouldn't add another variable because it won't be used elsewhere.

Reply:
@mariajgrimaldi So we discard all this?